Content.Shared/_CM14/Comtech/Barbed/Components/BarbedComponent.cs

```diff
@@ -0,0 +1,41 @@
+using Content.Shared.Damage;
+﻿using Robust.Shared.Serialization;
+using Content.Shared.DoAfter;
+
+namespace Content.Shared._CM14.Comtech.Barbed.Components
+{
+    [RegisterComponent]
+    public sealed partial class BarbedComponent : Component
+    {
+        [DataField(required: true)]
+        public DamageSpecifier ThornsDamage = default!;
+
+        [DataField("isBarbed")]
+        public bool IsBarbed = false;
+
+        // [DataField]
+        // public EntProtoId? Spawn; todo spawn a metal rod when wirecut
+
+        [DataField("wireTime")]
+        public float WireTime = 2.0f;
+
+        [DataField("cutTime")]
+        public float CutTime = 1.0f;
+    }
+
+    [NetSerializable, Serializable]
+    public enum BarbedWireVisuals : byte
+    {
+        Wired,
+    }
+}
```

Comment:
You don't need to specify the data field name, since it matches the field name (camelCased).
Also, with this being in shared, it should be getting networked to clients.
I don't think theres any danger with giving clients this information so it should be fine to send unconditionally.
This should also have an access attribute so that it fields can only be modified by BarbedSystem.

```suggestion
using Content.Shared._CM14.Comtech.Barbed;
using Content.Shared.Damage;
using Content.Shared.DoAfter;
using Robust.Shared.GameStates;
﻿using Robust.Shared.Serialization;

namespace Content.Shared._CM14.Comtech.Barbed.Components
{
    [RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
    [Access(typeof(BarbedSystem))]
    public sealed partial class BarbedComponent : Component
    {
        [DataField(required: true)]
        public DamageSpecifier ThornsDamage = default!;

        [DataField]
        public bool IsBarbed = false;

        // [DataField]
        // public EntProtoId? Spawn; todo spawn a metal rod when wirecut

        [DataField]
        public float WireTime = 2.0f;

        [DataField]
        public float CutTime = 1.0f;
    }

    [NetSerializable, Serializable]
    public enum BarbedWireVisuals : byte
    {
        Wired,
    }
}
```